recipes/cudadecon/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cudaDecon" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/scopetools/cudaDecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 29182331bb48d37b415d50365099314af9c824059cd6e9bc687e6d403d699e8e
+
+build:
+  number: 0
+  skip: True  # [osx or (cuda_compiler_version in ('None', 'undefined'))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - boost-cpp
+    - libtiff
+    - fftw
+  run:
+    - boost-cpp
+    - _openmp_mutex
```

Comment:
This is wrong. See our docs on openmp

Reply:
That was actually to satisfy an error message that was roughly something like "possible overlinking or runtime dependency not present". It's not clear from https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp that adding `libgomp` to the build deps does anything more than making something already implicitly included explicitly included instead (in which case, I'd still get that error). Maybe I'm missing something else? Will go ahead and try it either way.

Reply:
Annnnd it worked :)